stable/luigi/templates/service.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+  annotations:
```

Comment:
Can you please wrap the annotations with an if clause in case the annotations values are empty. You can see an example at https://github.com/helm/charts/blob/bdceb4ad926b9a69114ebb42d55ad87fb616707b/stable/janusgraph/templates/service.yaml#L10